src/app/account/signup/page.tsx

```diff
@@ -1,8 +1,13 @@
 import { SignUp } from '@/components/signup'
 import { pageStyles } from '@/styles/common'
 import { Container } from '@/styles/generated/jsx'
+import { PROD_ENV } from '@/server/config'
 
 export default function SignUpPage() {
+    // https://openstax.atlassian.net/browse/OTTER-107
+    // Temporarily remove signup page on production
+    if (PROD_ENV) return null
```

Comment:
PROD_ENV is a bit misnamed, it really means 'any environment other than local dev', it also includes dev, staging etc.  I still think this is fine though

Reply:
@chrisbendel I think we also need to hide the link to sign up that appears on the login page, right?